indico/queries/usermetrics.py

```diff
@@ -0,0 +1,113 @@
+from typing import List
+
+from indico.client.request import (
+    GraphQLRequest,
+    PagedRequest,
+)
+from indico.types import BaseType
+from indico.types.user_metrics import UserSummary, UserSnapshot
+
+
+class _PagedUserSnapshots(BaseType):
+    """
+    Class to hold paged snapshot data to make parsing easier.
+    """
+    results: List[UserSnapshot]
+
+
+class _UserMetricsFilters(BaseType):
+    """Filters for fetching user metric data."""
+    date: str
+    user_id: str
+    user_email: str
```

Comment:
Unfortunately can't link directly but [bullet point 7 under Graphql Schema](https://indicodata.atlassian.net/wiki/spaces/BE/pages/1352892435/User+Reports#Graphql-Schema)
user_id is a filter of type List[int] and user_email is a filter of type List[str]
so that folks can say give me userId=[1, 2, 3] - results for users where id=1 or 2 or 3